src/plugin-sidebar.js

```diff
@@ -172,7 +35,45 @@ const CreateBlockThemePlugin = () => {
 				icon={ tool }
 				title={ __( 'Create Block Theme' ) }
 			>
-				<ExportTheme />
+				<NavigatorProvider initialPath="/">
+					<NavigatorScreen path="/">
+						<PanelBody>
+							<VStack>
+								<NavigatorButton path="/save" icon={ edit }>
+									{ __( 'Save Changes…' ) }
```

Comment:
Initially I envisioned multiple "save" options where "Save as a Variation" was one of the options.  I believe that is targeted for Gutenberg itself so perhaps adding that here isn't helpful, but it might be.

I had also considered making individual parts save-able (such as just templates or just global styles).  Perhaps as check boxes or something?  A combination of expectation of eventual expansion of options as well as consistency of presentation (all options at the top level drill down rather than execute an option) were how we would up with two panels that do little.